.github/workflows/DeployBlazorClient.yml

```diff
@@ -22,24 +17,13 @@ jobs:
         id: builddeploy
         uses: Azure/static-web-apps-deploy@v1
         with:
-          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_NICE_FLOWER_068AD3503 }}
+          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN }}
           repo_token: ${{ secrets.GITHUB_TOKEN }} # Used for Github integrations (i.e. PR comments)
           action: "upload"
           ###### Repository/Build Configurations - These values can be configured to match your app requirements. ######
           # For more information regarding Static Web App workflow configurations, please visit: https://aka.ms/swaworkflowconfig
+          deployment_environment: Test

```

Comment:
I believe this will deploy everything always to the `Test` environment (see [named environments docs](https://learn.microsoft.com/en-gb/azure/static-web-apps/named-environments?tabs=github-actions)) even when it's the main branch or we're manually triggering it.
The original workflow creates an [environment per PR and then closes it afterwards](https://learn.microsoft.com/en-gb/azure/static-web-apps/build-configuration?tabs=github-actions#build-configuration), this may be better?